journal/modernized/18-chapter-17.adoc

```diff
@@ -191,11 +193,12 @@ We abode some days at Robert Haydock`'s, and had a meeting,
 to which Friends came out of the country.
 From there we travelled to Chester, and had a meeting on first-day,
 many Friends of the country coming to it.
+//In the PDF, the last part of the line reads "came" to it, not "coming" to it. Are those changes you want me to make if I happen upon them? I only saw the difference because it couldn't find that line in the original...I finally realized that's why some parts can't be found online.
```

Comment:
So...I figured out you are looking at a different google PDF than the one I used to create this book.  I took the text from the original Friends Library, and it DOES say _coming_ here. But there is also another edition of his journal online published in 1837 where this has been changed to _came_. If you see something like this, and it doesn't sound right, feel free to change it in the modernized version.